project/Build.scala

```diff
@@ -105,7 +105,12 @@ object DottyBuild extends Build {
       (dottyOrganization % "dotty-sbt-bridge" % scalaVersion.value % "component").sources(),
 
     // sbt gets very unhappy if two projects use the same target
-    target := baseDirectory.value / ".." / "out" / name.value
+    target := baseDirectory.value / ".." / "out" / name.value,
+
+    // The non-bootstrapped dotty-library is not necessary when bootstrapping dotty
+    autoScalaLibrary := false,
```

Comment:
I assume there's some magic in sbt or the dotty-sbt-bridge that causes `autoScalaLibrary := true` to depend on `dotty-library` (instead of `scala-library`). Otherwise the comment doesn't make any sense. Am I right?